src/d3a/setup/jira/d3asim_1637.json

```diff
@@ -113,16 +109,15 @@
               "initial_buying_rate": 0.0,
               "use_market_maker_rate": true,
               "daily_load_profile_uuid": null,
-              "energy_rate_increase_per_update": null,
               "display_type": "Load"
             },
             {
               "name": "PV 3",
-              "type": "PredefinedPV",
+              "type": "PV",
               "uuid": "f04cf867-eeac-48a0-89d2-23695340e268",
               "libraryUUID": null,
               "panel_count": 3,
-              "fit_to_limit": false,
```

Comment:
SO you moving from setting the energy_rate change per update to "fit_to_limit" . Any reason for this?

Reply:
Reverting it to original state, and actually the original json setup file revealed an issue now that `Storage` strategy was not using `energy_rate_increase_per_update`, `energy_rate_decrease_per_update` since they were not part of parameters. Have added it in storage strategy parameters in my latest commit. Thanks for nitpicking.